src/nodewatcher/plugins/slurm.py

```diff
@@ -99,10 +101,10 @@ def is_node_down():
 
 
 def _get_node_slots():
-    hostname = check_command_output("hostname")
+    hostname = socket.get_hostname()
     # retrieves number of slots for a specific node in the cluster.
     # Output format:
     # 4
-    command = "/opt/slurm/bin/sinfo -o '%c' -n {0} -h".format(hostname)
+    command = "sinfo -o '%c' -n {0} -h".format(hostname)
```

Comment:
Hmm if for other distros "slurm" is installed from source and ends up in `/opt` then this is a breaking change as `/opt` is usually not on the PATH. Meaning the code either has to deal with multiple locations in some way or `/opt/slurm/bin` has to be added to PATH. Similar problems in other areas.

Reply:
Distributors shouldn't install anything in `/opt`. By removing the path to the binary we make the assumption that it is in the search patch, so if the binaries were installed in a non standard directory, it should be ensured that the service process has that to its search path added IMHO.

It seems that the `aws-parallelcluster` orchestration adds `/opt/slurm/bin` to the system wide search path, so this change shouldn't break anything.